modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc

```diff
@@ -120,82 +120,24 @@ image::ms-dynamics/ms-service-bus-windows-1.png[Connection]
 |Skip connectivity test |In case you have limited access to Windows Service Bus resources and you want to skip the connectivity test performed at startup you need to set this setting to true.
 |===
 
-=== Sources
+=== Connector-Specific Input Source
```

Comment:
Why "Connector-Specific Input Source"?

Reply:
Because you can also use HTTP Listener or Scheduler with this connector.

Reply:
You can use the HTTP Listener, Scheduler, or any source alongside any connector. In the context of the documentation of a specific connector, this title just seems needlessly obfuscated in my opinion. But if it is consistent with the rest of the documentation, I'm fine with this. 👍